mlbench_core/lr_scheduler/pytorch/lr.py

```diff
@@ -238,7 +239,7 @@ def f(self, iteration):
         return 1 / max(1, (self.shifting_param + iteration))
 
 
-class SGDLR(LambdaLR):
+class TimeDecayLR(LambdaLR):
```

Comment:
needs to be renamed in __init__.py as well, otherwise unittests fail

Reply:
Done, thanks.